ballerina/tests/csv_io.bal

```diff
@@ -625,6 +753,39 @@ function testFileReadCsvAsStreamUsingResourceFile() returns error? {
     test:assertEquals(i, 3);
 }
 
+@test:Config {dependsOn: [testFileWriteCsvFromStreamUsingResourceFileRecords]}
+function testFileReadCsvAsStreamUsingResourceFileRecords() returns error? {
+    string filePath = TEMP_DIR + "workers4_A.csv";
+    string[][] expectedContent = [
+        ["Anne Hamiltom", "Software Engineer", "Microsoft", "26 years", "New York"],
+        [
+            "John Thomson",
+            "Software Architect",
+            "WSO2",
+            "38 years",
+            "Colombo"
+        ],
+        [
+            "Mary Thompson",
+            "Banker",
+            "Sampath Bank",
+            "30 years",
+            "Colombo"
+        ]
+    ];
+    stream<Employee5, Error?> result = check fileReadCsvAsStream(filePath);
+    int i = 0;
+    check result.forEach(function(Employee5 val) {
+        int j = 0;
+        foreach string s in val.keys() {
+            test:assertEquals('string:trim(<string>val.get(s)), expectedContent[i][j]);
```

Comment:
It gave following error when tried to use `ERROR [tests/csv_io.bal:(822:44,822:54)] incompatible types: expected 'string', found 'anydata'`. I'll check for the cause and alternatives.

Reply:
I think the reason you need an explicit cast here is because `Employee5` is an open record. You shouldn't need this cast if `Employee5` is a closed record (i.e `{| ... |}` instead of `{ ... }`)

Reply:
@heshanpadmasiri Yes, making it a closed record solved the issue.